### DISCUSSION TITLE
ReferenceError: TouchEvent is not defined

### DISCUSSION THREAD

**Discussion by tomhusband** [blakeblackshear/frigate#6974] [General]
I'm getting dozens of these errors recently and am not sure to look to resolve. Any hints appreciated. Many thanks.

`2023-07-01 09:42:23.749 ERROR (MainThread) [frontend.js.latest.202306070] http://xxx.xxx.x.xxx:8123/hacsfiles/frigate-hass-card/zoomer-dc22c08a.js:1:11286 ReferenceError: TouchEvent is not defined`

**Comment by NickM-27** [comment]
this is the wrong repo, that is related to the frigate card
